app/app.go

```diff
@@ -290,6 +298,7 @@ type Evmos struct {
 	VestingKeeper    vestingkeeper.Keeper
 	RecoveryKeeper   *recoverykeeper.Keeper
 	RevenueKeeper    revenuekeeper.Keeper
+	IbcEvmTxKeeper   ibcevmtxkeeper.Keeper
```

Comment:
@GAtom22 In what scenario does the ibc evm module account need to receive tokens and hold a balance? 

also, nit
```suggestion
	IBCEvmKeeper   ibcevmkeeper.Keeper
```